Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CertficateBean.java

```diff
@@ -266,6 +272,58 @@ private void importSelectedCertificate() {
         }
     }
 
+    public void updateSelectedCertificate() {
+        LOG.info("updateSelectedCertificate");
+        if (selectedTSCertificate != null){
+            String alias = selectedTSCertificate.getAlias();
+            if (StringUtils.isNotBlank(alias) && !service.isAliasInUse(alias, service.fetchTrustStores())) {
+                try {
+                    boolean updateStatus = service.updateCertificateTS(oldAlias, selectedTSCertificate);
```

Comment:
Use inline method call like this: postUpdateAction(service.updateCertificateTS(oldAlias, selectedTSCertificate), VIEW_CERT_ERR_MSG);